grails-app/assets/javascripts/postal-code-utils.js

```diff
@@ -0,0 +1,64 @@
+function PostalCodeUtils() {
+    var _this = this;
+    _this.reference = $(".js-person-address-form");
+    _this.postalCodeInput = _this.reference.find(".js-postal-code");
```

Comment:
```suggestion
    _this.postalCode = _this.reference.find(".js-postal-code");
```
Deixa para indicar o tipo de campo quando for estritamente necessário